tests/posix/mt/mt.cpp

```diff
@@ -38,16 +38,33 @@
 
 #include "pmemfile_test.hpp"
 
+static int ops = 20;
+
+static PMEMfilepool *global_pfp;
+
 class mt : public pmemfile_test {
 public:
-	mt() : pmemfile_test()
+	long ncpus;
+	pthread_t *threads;
+
+	mt() : pmemfile_test(256 << 20)
 	{
+		ncpus = sysconf(_SC_NPROCESSORS_ONLN);
+		threads = new pthread_t[(size_t)ncpus * 2];
```

Comment:
Could we use C++ standard library threads in tests instead?
There is also http://en.cppreference.com/w/cpp/thread/thread/hardware_concurrency
That would make windows porting far easier as well.